dev/packages/alpha/cisco-0.0.2/dataset/asa/manifest.yml

```diff
@@ -0,0 +1,59 @@
+title: Cisco ASA logs
+release: ga
+type: logs
+streams:
+  - input: syslog
+    title: Cisco ASA logs
+    description: Collect Cisco ASA logs
+    template_path: stream.yml
+    vars:
+      - name: tags
+        type: text
+        title: Tags
+        multi: true
+        required: true
+        show_user: true
+        default:
+          - cisco-asa
+      - name: syslog_port
+        type: integer
+        title: Syslog Port
+        multi: false
+        required: true
+        show_user: true
+        default: 9001
+      - name: log_level
+        type: integer
+        title: Log Level
+        multi: false
+        required: true
+        show_user: true
+        default: 7
+  - input: file
+    title: Cisco ASA logs
+    description: Collect Cisco ASA logs from file
+    template_path: file.yml
+    vars:
+      - name: paths
+        type: text
+        title: Paths
+        multi: true
+        required: true
+        show_user: true
+        default:
+          - /var/log/cisco-asa.log
+      - name: tags
+        type: text
+        title: Tags
+        multi: true
+        required: true
+        show_user: true
+        default:
+          - cisco-asa
+      - name: log_level
+        type: integer
+        title: Log Level
+        multi: false
+        required: true
+        show_user: true
```

Comment:
Same here: is it required to present it to user? I think screenshots will help us understand which fields are required.